### PR TITLE
Rework bounties

### PR DESCRIPTION
**Pull request by soc1c**

### CODE REVIEW

File: bounties.md [hunk 1]
```diff
@@ -1,51 +1,41 @@
...
--   **Sponsor Prizes:** Some of ETHBerlin’s sponsors have their own price awards. They can each pay out a maximum of TBD €. These prizes will be funded and distributed via Gitcoin. Note that ETHBerlin has no influence on neither the challenge nor the judging, this is entirely up to the sponsors. You can find all the prize awards [here](https://github.com/ethberlinzwei/Bounties/issues).
--   **Community Improvement Awards:** These are small prizes endowed with TBD € each by ETHBerlin. Check them out [here](https://github.com/ethberlinzwei/Bounties).
+This year, there are different prize categories available.
 
-**IMPORTANT:** You can submit your work to multiple prize awards, they are not exclusive! For example you can submit your work to the open track bounty and a sponsor specific prize, just make sure to submit it everywhere where you want to participate.
+-   **Open Track Prize:** This is the main prize of ETHBerlin Zwei. There will be 10 winners. The grand bounty is of approximately 0,00 € and it’s distributed equally. You can calculate it ;)
```
Comment (ligi) [inline bounties.md:17]: "the grand bounty is of approximately 0,00 € and it’s distributed equally."

looks off to me

Comment (soc1c) [reply bounties.md:17]: ref https://github.com/ethberlinzwei/KnowledgeBase/issues/39

File: judges.md [hunk 2]
```diff
@@ -1 +1,20 @@
+`<<` Back to [Getting Help](./mentors.md) | Continue to [Sponsors](./sponsors.md) `>>`
+
+---
+
 # Judges and Judging
```
Comment (ligi) [inline judges.md]: would signal to change to Judging (Judges are part of Judging process) and rename the document from judges to judging

File: bounties.md (hunk 1 above)
Comment (ligi) [reply bounties.md:17]: yes - I do not mean the amount - I meant the sentence structure. also perhaps we should add TODO instead of 0.00

**Review by ligi — CHANGES_REQUESTED** [review]
let's rename judges.md to judging.md and replace 0.00 with TBD

**Comment by soc1c** [comment]
renaming the file means we need to fix all links. I have to jump on a call in 5 minutes and really need a break. IMO both file name and TBD doesnt matter much. we have tracking tickets and need to update it anyways. "0" reads better than "tbd"

**Review by ligi — APPROVED** [review]

**Comment by ligi** [comment]
fair enough

**Pull request merged**